syntax = "proto3";
package proto;
option go_package = "example/;example";

message OrderRequest{
  string order_id = 1 ;
  int64 timestamp = 2;
}

message OrderResponse{
  string order_id = 1;
  string order_name = 2;
  string order_status = 3;
}

service OrderService{
  rpc GetOrder(OrderRequest) returns (OrderResponse);
}